DirectEmployers/MyJobs
# [PD-3674] refactor sidebar dimension component

## What's this PR do?
Refactors the sidebar dimension component to be a functional component instead of class

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application

## How should this be manually tested?
View the code OR pull down the repository and go into any analytics application and make sure the sidebar is still functional

## Any background context you want to provide?
